src/AnnotationData.php

```diff
@@ -24,4 +24,25 @@ public function keys()
     {
         return array_keys($this->getArrayCopy());
     }
+
+    public function set($key, $value = '')
+    {
+        $this->offsetSet($key, $value);
+        return $this;
+    }
+
+    public function append($key, $value = '')
+    {
+        $data = $this->offsetGet($key);
+        if (is_array($data)) {
+            $this->offsetSet($key, array_merge($data, $value));
+        }
+        elseif (is_numeric($data)) {
+            $this->offsetSet($key, $data + $value);
```

Comment:
I don't think that `append` should ever do addition. It seems unlikely that anyone would ever want do this on purpose, and this could be the source of a very obscure bug if some data the user thought was a string was interpreted as a numeric value.

Reply:
That's fair. I was going for completeness.